src/main/kotlin/mate/academy/service/RandomFigureType.kt

```diff
@@ -0,0 +1,45 @@
+package mate.academy.service
+
+import mate.academy.figure.Circle
+import mate.academy.figure.Figure
+import mate.academy.figure.IsoscelesTrapezoid
+import mate.academy.figure.Rectangle
+import mate.academy.figure.RightTriangle
+import mate.academy.figure.Square
+import kotlin.random.Random
+
+private const val MAX_RANDOM_VALUE = 20.0
+
+enum class RandomFigureType {
```

Comment:
Interesting solution, however by using enum assumed that it contains just figure's type.
Adding additional functionality to enum reduce readability and flexibility of the code.

Let's check how it could be simplified.

- FigureType enum contains just figures CIRCLE , ISOSCELES_TRAPEZOID ...
- In getRandomFigure() when() construction handle generating random figure

smth like this
when(YOUR_ENUM.values().random()){
CIRCLE -> ...
ISOSCELES_TRAPEZOID  -> ...
....
}